tests/test_suites/ShortSystemTests/test_custom_goal_migration_during_delay_disconnect.sh

```diff
@@ -40,7 +40,11 @@ saunafs setgoal raid_goal "${info[mount0]}"/* > /dev/null
 
 # Chunks should be replicated only on 'raid' server.
 # Replication to other servers is delayed because of disconnected chunkserver 2.
-assert_eventually_prints 20 'find_chunkserver_metadata_chunks 0 | wc -l' '5 seconds'
+if is_windows_system; then
+	assert_eventually_prints 20 'find_chunkserver_metadata_chunks 0 | wc -l' '25 seconds'
+else
+	assert_eventually_prints 20 'find_chunkserver_metadata_chunks 0 | wc -l' '5 seconds'
+fi
```

Comment:
Similar here, the replication speed should only be affected like this by the master and chunkservers configuration, not the client.

Reply:
This is the same issue with speed as mentioned here: https://github.com/leil-io/saunafs/pull/24#discussion_r1522690499.